presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -763,6 +764,35 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
         return TableStatistics.empty();
     }
 
+    @Override
+    public void createSchema(JdbcIdentity identity, String schemaName)
+    {
+        String sql = "CREATE SCHEMA " + quoted(schemaName);
+
+        try (Connection connection = connectionFactory.openConnection(identity)) {
+            execute(connection, sql);
+        }
+        catch (SQLException e) {
+            throw new PrestoException(JDBC_ERROR, e);
+        }
+    }
+
+    @Override
+    public void dropSchema(JdbcIdentity identity, String schemaName)
+    {
+        if (!getTableNames(identity, Optional.of(schemaName)).isEmpty()) {
```

Comment:
Schema may contain other objects (not only tables and views). Also schema could be used in other objects (like being used in view definition) and so underlying database could prevent removal of the schema here. In general it looks we are not able to verify if schema is safe to be removed.  Hence I would remove this check.